nutridine/src/screens/Homepage.js

```diff
@@ -0,0 +1,92 @@
+import React, { useState, useEffect } from 'react';
+import { Box, Input, useBreakpointValue, useColorMode, Text } from '@chakra-ui/react';
+
+import FilterGroup from '../components/FilterGroup';
+import FoodCardList from '../components/FoodDisplay/FoodCardList';
+import useMealsByQuery from '../hooks/Meals/useMealsByQuery';
+
+function Homepage() {
+    const { colorMode } = useColorMode();
+    const filters = ['Burger', 'Sushi', 'Pizza', 'Chicken'];
```

Comment:
Are you planning on adding features such as `high-protein` `low-calories`..etc that will require an updated query to fetch based on the nutritional facts?

Reply:
if so you should be able to use the `useMealsByMacros` hook to grab that

Reply:
Oh right, yes, i'll probably do that.

Reply:
Okay nice, I see it now! Gave it a try as well and it seems to work like a charm